tabpy-server/server_tests/test_config.py

```diff
@@ -57,18 +57,19 @@ def test_no_state_ini_file_or_state_dir(self, mock_os, mock_file_exists, mock_pa
 class TestPartialConfigFile(unittest.TestCase):
 
     def setUp(self):
-        config_file = '../tabpy_server/common/config.py'
+        config_file = os.path.join(os.path.dirname(os.path.dirname(__file__)),
```

Comment:
FYI if there's a plan to drop Py27 in the roadmap (it gets officially dropped Jan 1, 2020) these types of path resolutions can be greatly simplified with the use of `pathlib`